andino_firmware/src/motor.cpp

```diff
@@ -63,6 +63,39 @@
 // OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-void initMotorController();
-void setMotorSpeed(int i, int spd);
-void setMotorSpeeds(int leftSpeed, int rightSpeed);
+#include "motor.h"
+
+#include "Arduino.h"
+
+namespace andino {
+
+void Motor::set_state(bool enabled) {
+  if (enabled) {
+    digitalWrite(enable_gpio_pin_, HIGH);
+  } else {
+    digitalWrite(enable_gpio_pin_, LOW);
+  }
+}
+
+void Motor::set_speed(int speed) {
+  bool forward = true;
+
+  if (speed < 0) {
+    speed = -speed;
+    forward = false;
+  }
+  if (speed > 255) {
+    speed = 255;
```

Comment:
Make 0 and 255 constant members in the class. We could at some point extend it to receive the limits from the constructor.

Reply:
Done.